pyaedt/application/Analysis.py

```diff
@@ -95,148 +89,122 @@ def Position(self):
+        """Coordinate system axis constant.
         
-        :return: Coordinate System Axis Constants Tuple (.X, .Y, .Z)
-
-        Parameters
-        ----------
-
         Returns
         -------
+        type
+            Coordinate System Axis Constants Tuple (.X, .Y, .Z).
 
         """
         return CoordinateSystemAxis()
 
     @property
     def CoordinateSystemPlane(self):
-        """CoordinateSystemPlane Constants
+        """Coordinate system plane constants.
         
-        :return: Coordinate System Plane Constants Tuple(.XY, .YZ, .XZ)
-
-        Parameters
-        ----------
-
         Returns
         -------
+        type
+            Coordinate system plane constants tuple(.XY, .YZ, .XZ).
 
         """
         return CoordinateSystemPlane()
 
     @property
     def View(self):
-        """Planes To check if redundant to CoordinateSystemPlane
-        
+        """Planes. (To check if redundant to CoordinateSystemPlane.)
+             
         
-        :return: Coordinate System Plane String Tuple("XY", "YZ", "XZ")
-
-        Parameters
-        ----------
-
         Returns
         -------
+        type
+            Coordinate System Plane String Tuple("XY", "YZ", "XZ").
 
         """
         return Plane()
 
     @property
     def GravityDirection(self):
-        """To check if redundant
+        """Gravity direction. (To check if redundant.)
         
-        :return: Gravity Direction Tuple (XNeg, YNeg, ZNeg, XPos, YPos, ZPos)
-
-        Parameters
-        ----------
-
         Returns
         -------
+        Gravity direction tuple (XNeg, YNeg, ZNeg, XPos, YPos, ZPos).
 
         """
         return GravityDirection()
 
     @property
     def modeler(self):
-        """:return: modeler object"""
+        """Modeler.
+        
+        Returns
+        -------
+        :class:`pyaedt.modeler.Modeler.Modeler`
+            Modeler object.
+        """
         return self._modeler
 
     @property
     def mesh(self):
-        """Mesh Object
+        """Mesh.
         
-        :return: mesh object
-
-        Parameters
-        ----------
-
         Returns
         -------
-
+        type
+            Mesh object.
         """
         return self._mesh
 
     @property
     def post(self):
-        """Post Processor Object
+        """PostProcessor.
         
-        :return: post processor object
-
-        Parameters
-        ----------
-
         Returns
         -------
-        :class:`pyaedt.modules.AdvancedPostProcessing.PostProcessor`
+        :class:`pyaedt.modules.PostProcessor.PostProcessor`
+            PostProcessor object.
         """
         return self._post
 
     @property
     def osolution(self):
-        """osolution Object
+        """Solution.
         
-        :return: Solutions Module object
-
-        Parameters
-        ----------
-
         Returns
         -------
+        :class:`pyaedt.modules.AdvancedPostProcessing.PostProcessor.post_osolution`
```

Comment:
```suggestion
        AEDT object
```
This is a call made to the AEDT application to get the Solutions object to then perform some operations. There really isn't a type for this object, so I wrote "AEDT object" but feel free to change it.